js/map_utils.js

```diff
@@ -185,10 +186,22 @@ function documentCategorySwitch(category, prevCategory, action) {
 }
 
 
+function highlightCircle(currentCircle) {
+  var orig = currentCircle,
+      origNode = orig.node();
+  var duplicate = d3.select(origNode.parentNode.appendChild(origNode.cloneNode(true), 
+                                                            origNode.nextSibling));
+                                                            
+  // style duplicated circles sitting on top
+  duplicate
+    .classed('circle-duplicate', true)
+    .style("pointer-events", "none")
+    .style("opacity", 1); // makes the duplicate circle on the top
+}
 
-function updateTitle(category) {
-  d3.select("#maptitle")
-    .text("Water Use Data for " + activeView + ", 2015, " + category);
+function unhighlightCircle() {
+  d3.select('.circle-duplicate')
+    .remove(); // delete duplicate
```

Comment:
The unhighlightCircle function should look pretty different from both of these. Should be: https://github.com/USGS-VIZLAB/water-use-15/blob/master/js/circles.js#L100-L105
```js
function unhighlightCircle() {

  map.select('circle#wu-highlight')
    .classed('hidden', true);

}
```